packages/rs-drive-abci/src/execution/validation/state_transition/processor/v0/mod.rs

```diff
@@ -109,7 +112,24 @@ pub(super) fn process_state_transition_v0<'a, C: CoreRPCLike>(
         }
     }
 
-    // Only identity update has advanced structure validation without state
+    // This is for identity credit withdrawal and identity credit transfers
```

Comment:
Please update the comment: It also validates min balance for other ST. It would be helpful if you describe there what this pre-check for